db/migrate/20170809153751_update_old_users_required_fields.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateOldUsersRequiredFields < ActiveRecord::Migration
+  def change
+    User.where(name: nil).update_all(name: "anonymous")
+    User.where(gender: nil).update_all(gender: "unknown")
```

Comment:
It might make sense to go even further with this and move all users that don't fit in one of our current bucks into 'unknown'. That should probably be a different PR though